simpledotorg/simple-server
# Add BS 200-299 or BS >=300 graph

**Story card:** [sc-7781](https://app.shortcut.com/simpledotorg/story/7781/add-bs-200-299-or-bs-300-graph)

## Because

We need add a graph showing the trend of BS 200-299 or BS >=300 patients on the diabetes reports page

## This addresses

- Region Summary and Region Summary Schema report blood sugar metrics
- Repository and Repository presenter present data to the controller
- The diabetes view adds html to display the BS 200-299 or BS >=300 graph

## Test instructions

1. Turn on `diabetes_management_reports` on flipper
2. Visit the reports page of a facility which has `enable_diabetes_management` set to true
3. Click on `Diabetes` link
4. You should see the BS 200-299 or BS >=300 graph
<img width="673" alt="Screenshot 2022-05-04 at 6 32 27 PM" src="https://user-images.githubusercontent.com/32141642/166686616-c5da977b-9624-4c08-8e8a-28fe7afd984e.png">

 <img width="666" alt="Screenshot 2022-05-04 at 6 30 21 PM" src="https://user-images.githubusercontent.com/32141642/166686342-cb7dd925-2884-40b7-945d-9cb16aa5173b.png">
